types/index.d.ts

```diff
@@ -47,7 +45,11 @@ declare function useVirtual<T>(options: {
   onScrollElement?: React.RefObject<HTMLElement>
   scrollOffsetFn?: (event?: Event) => number
   rangeExtractor?: (range: Range) => number[]
-}): {
+}
+
+declare function defaultRangeExtractor(range: Range): number[]
+
+declare function useVirtual<T>(options: ReactVirtualOptions): {
```

Comment:
Pass generic param to `Options`

```suggestion
declare function useVirtual<T>(options: Options<T>): {
```

Reply:
updated!